.drone.yml

```diff
@@ -6,7 +6,7 @@ pipeline:
       - ln -s /var/cache/drone/scala-scala scala-scala
       - ln -s /var/cache/drone/ivy2 "$HOME/.ivy2"
       - ./scripts/update-scala-library
-      - sbt -J-Xmx4096m -J-XX:ReservedCodeCacheSize=512m -J-XX:MaxMetaspaceSize=1024m -Ddotty.drone.mem=4096m "${TEST}"
+      - sbt -J-Xmx24G -J-XX:ReservedCodeCacheSize=512m -J-XX:MaxMetaspaceSize=4G -Ddotty.drone.mem=4096m "${TEST}"
```

Comment:
`dotty.drone.mem` is used in `Build.scala`, it's the amount of memory we give to each jvm started from sbt.